x-pack/plugins/threat_intelligence/public/modules/cases/components/add_to_new_case/add_to_new_case.tsx

```diff
@@ -57,9 +59,11 @@ export const AddToNewCase: VFC<AddToNewCaseProps> = ({
     attachmentMetadata
   );
 
-  // disable the item if there isn't an indicator name
+  // disable the item if there is no indicator name or if the user doesn't have the right permission
   // in the case's attachment, the indicator name is the link to open the flyout
-  const disabled: boolean = attachmentMetadata.indicatorName === EMPTY_VALUE;
+  const invalidIndicatorName: boolean = attachmentMetadata.indicatorName === EMPTY_VALUE;
+  const hasCreatePermission: boolean = permissions.create && permissions.read && permissions.push;
```

Comment:
@cnasikas I'm not entirely sure that these are the permissions required for the add to new case action. Actually I couldn't find an explanation of what each permission means. Is there a place where these are documented? I'm specifically confused about the difference between `update` and `push`.

Reply:
> I'm specifically confused about the difference between update and push

Yeah our documentation needs to be improved in this area. `update` means any modification of an existing entity, like changing a case title, or editing a comment. `push` refers to sending a case to an external service like Jira, or SerivceNow. I'll look over the code more tomorrow but my guess is you can ignore if the user has `push` permissions.

If you are adding an attachment to an existing case we'd want the user to have `update` permissions as well as `create`.

Reply:
thank you both!